ads/3p.md

```diff
@@ -0,0 +1,99 @@
+```
+<amp-brightcove
+      data-account="906043040001"
+      data-video-id="1401169490001"
+      data-player-id="180a5658-8be8-4f33-8eba-d562ab41b40c"
+      layout="responsive" width="480" height="270">
+  </amp-brightcove>
+```
+For instructions on how to develop an amp tag like brightcove, please see [amp-brightcove.js](https://github.com/ampproject/amphtml/blob/master/extensions/amp-brightcove/0.1/amp-brightcove.js)
+
+The other popular option is for video players to take advantage of the amp-iframe (https://github.com/ampproject/amphtml/blob/master/extensions/amp-iframe/amp-iframe.md) extension (with post frame or fallback markup) and serve the player within the iframe. 
+
+#### Video Ad Network
+
+*Examples : Tremor, Brightroll*
+
+If you are a video ad network, please work with your publisher to ensure that :
+
+-All video assets are being served over HTTPS
+-The publisher’s video player has AMP support
+
+
+#### Data Management Platform (DMP)
+*Examples : KRUX, Bluekai*
+
+[See how to enhance custom ad configuration](https://github.com/ampproject/amphtml/blob/master/builtins/amp-ad.md#enhance-incoming-ad-configuration)
+
+You can use a similar approach to enrich the ad call by passing in audience segments that you get from the user cookie into the ad call.
+
+#### Viewability Provider
+
+*Examples : MOAT, Integral Ad Science*
+
+Viewability providers typically integrate with publishers via the ad server’s creative wrappers. If that is the case, ensure that the creative wrapper loads all assets over HTTPS.
+
+For e.g. for MOAT, make sure http://js.moatads.com is switched to  https://z.moatads.com
+
+If you instead would like to directly embed viewability JS on the publisher page, use the approach to inject javascript on a remote server. [See how to enhance custom ad configuration](https://github.com/ampproject/amphtml/blob/master/builtins/amp-ad.md#enhance-incoming-ad-configuration)
+
+Alternatively, see appraoch to using the [intersection obeserver pattern](#ad-viewability)
+
+#### Content-Recommendation Platform
+
+*Examples : Taboola, Outbrain*
+
+As content recommendation platforms, if you have some piece of Javascript embeded on the publisher website today but the approach will not work in AMP pages. If you would like to recommend content via the publisher’s ad server, we suggest that you use the amp-embed extension to request the content details. (served like the creative).
+
+Instead, you can also create your own AMP extension tag which will allow you to embed a tag like <amp-taboola> in the publisher’s page instead of custom javascript.
+
+
+If instead you’d like to simply pass in the elements of the content you’d like to recommend to the publisher’s page (so the publisher can maintain the look and feel), use the approach to inject javascript on a remote server. [See how to enhance custom ad configuration](https://github.com/ampproject/amphtml/blob/master/builtins/amp-ad.md#enhance-incoming-ad-configuration)
```

Comment:
I don't fully get this. Are we suggesting publishers to put some CSS into their remote.html? Not the worst idea :) We should share that with taboola and outbrain.